tests/browser/dynamic/style-override.tests.ts

```diff
@@ -48,4 +48,160 @@ describe('Style override', () => {
         await expect(page.evaluate(() => getComputedStyle(document.querySelector('h1')).color)).resolves.toBe('rgb(232, 230, 227)');
         await expect(page.evaluate(() => getComputedStyle(document.querySelector('h1 strong')).color)).resolves.toBe('rgb(255, 26, 26)');
     });
+
+    it('should restore override', async () => {
+        await loadTestPage({
+            '/': multiline(
+                '<!DOCTYPE html>',
+                '<html>',
+                '<head>',
+                '   <script>',
```

Comment:
I would suggest to use `page.evaluate()` where possible, e.g.
```javascript
await page.evaluate(() => {
    const styleElement = document.createElement("style");
    styleElement.classList.add("testcase-style");
    document.head.append(styleElement);
    styleElement.sheet.insertRule("h1 { color: gray } ")
    styleElement.sheet.insertRule("strong { color: red } ")
})`
```

Please note that the function passed as an argument is converted to a string, and is executed in the page using `eval()`, follow the docs if you need to pass something into this function.

Don't forget about `code-style`

Reply:
Oh woops. That was something for testing purposes...